### PR TITLE
podman: Fix kernel downgrade

### PR DESCRIPTION
**Pull request by cfergeau**
This currently fails with dependency issues related to 
missing kernel-uname-r dependencies. 
This PR should fix that. During my testing, first try failed, and second
attempt succeeded, so this must go through CI first :)

Fixes: https://github.com/crc-org/snc/issues/700

### CODE REVIEW

File: createdisk-library.sh [hunk 1]
```diff
@@ -193,7 +193,11 @@ function downgrade_kernel() {
 	    ;;
     esac
 
-    ${SSH} core@${vm_ip} "sudo rpm-ostree override -C replace ${bodhi_url}"
+    ${SSH} core@${vm_ip} -- 'sudo sed -i -z s/enabled=0/enabled=1/ /etc/yum.repos.d/fedora.repo'
+    ${SSH} core@${vm_ip} -- 'sudo sed -i -z s/enabled=0/enabled=1/ /etc/yum.repos.d/fedora-updates.repo'
+    ${SSH} core@${vm_ip} "sudo rpm-ostree override replace ${bodhi_url}"
```
Comment (praveenkumar) [inline createdisk-library.sh:198]: for consistency can use `--` here also `${SSH} core@${vm_ip} -- "sudo .."`

**Comment by praveenkumar** [comment]
>This PR should fix that. During my testing, first try failed, and second
attempt succeeded, so this must go through CI first :)

CI only create the bundle for podman and doesn't run it using CRC so may be even CI is green, created bundle need manual testing. I will create an issue to atleast test that created bundle on linux.

**Comment by cfergeau** [comment]
> > first try failed, and second
> 
> Can you be more specific what/how/when this failed? The resulting image failed on start up the first time, but a restart worked? or was a new build needed?

This was a failure during build, but after investigating and trying a second build, it worked. CI is green now, so this seems fine.

File: createdisk-library.sh (hunk 1 above)
Comment (cfergeau) [reply createdisk-library.sh]: Force-pushed some changes to address this, but they are untested, need to wait for CI.

Comment (cfergeau) [reply createdisk-library.sh]: CI is green.

**Comment by praveenkumar** [comment]
/lgtm
/approve


Tested with https://github.com/crc-org/snc/pull/704

**Pull request merged**